src/mlpack/methods/neighbor_search/ns_model.hpp

```diff
@@ -13,12 +13,24 @@
 #include <mlpack/core/tree/binary_space_tree.hpp>
 #include <mlpack/core/tree/cover_tree.hpp>
 #include <mlpack/core/tree/rectangle_tree.hpp>
-
+#include <boost/variant.hpp>
```

Comment:
Ok, looks like there is nothing special we need to do with our CMake configuration for this change then.